### ISSUE TITLE
New page for displaying contributors in nextjs

### ISSUE THREAD

**Issue by jsvigneshkanna** [jsvigneshkanna/tailwind_ui_components#182]
### Feature Title

Contributors page

### Description

Need to create a new page named "contributors" which should have a navbar, contributors component and footer

The new contributor component should get this repo contributor to display pictures and name/ GitHub link and display them as a card
Follow the footer background colour gradient for this new component

The styling should be done only in tailwindcss and should be mobile responsive

Make it minimalist and we can improve it further 

### Motivation

This repo is solely built by open-source contributors, we need to display them on our website

### Issue Type

Frontend

### Screenshot of the feature if done?

![image](https://user-images.githubusercontent.com/42484705/197320039-ec5cffe0-9dba-4b6a-b5c5-028fcd487c00.png)

### Are you taking this feature implementation

No - up for grabs

**Comment by AnnieDebbie** [comment]
Hello
I would love to work on this issue. Please assign it to me

**Comment by jsvigneshkanna** [comment]
@AnnieDebbie great 
Thanks for checking this repo, creativity on design is upto you but please follow the color palette alone

**Comment by jsvigneshkanna** [comment]
@AnnieDebbie ,

let me know if any help needed

**Comment by AnnieDebbie** [comment]
Hello @jsvigneshkanna 
Thank you so much for reaching out. 
No help needed yet. 
I just needed to catch up with the code base, using tailwind and creating a suitable design. 
I should be done latest tomorrow. I hope that's okay?

**Comment by jsvigneshkanna** [comment]
Sure @AnnieDebbie, take your time :)

**Comment by Viren0602** [comment]
Hello sir, can you assign this issue to me?

**Comment by jsvigneshkanna** [comment]
Sure @Viren0602 
Please star this repo

**Comment by AnnieDebbie** [comment]
@jsvigneshkanna 
I know I had promised to send today but I moved really slow because I came down with a cold. I have attached screenshots of what I've done so far for the desktop view. 
Regarding the blogger icon, I could only the exact icon on fonts awesome and I don't know how you feel About me using that.
![Screenshot from 2022-10-24 22-15-28@2x](https://user-images.githubusercontent.com/32017636/197634441-91a90e21-ad21-4da3-9a37-b4470e41c962.png)
